src/dash-table/components/Table/style.ts

```diff
@@ -0,0 +1,13 @@
+import { library } from '@fortawesome/fontawesome-svg-core';
+import { faEyeSlash, faTrashAlt } from '@fortawesome/free-regular-svg-icons';
+import { faEraser, faPencilAlt, faSort, faSortDown, faSortUp } from '@fortawesome/free-solid-svg-icons';
+
+library.add(
+    faEraser,
+    faEyeSlash,
+    faPencilAlt,
+    faSort,
+    faSortDown,
+    faSortUp,
+    faTrashAlt
+);
```

Comment:
Loads font-awesome icons in a way that allows for tree-shaking. The downside is that this file needs to be updated when new icons are needed.